runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -2843,7 +2843,7 @@ remoteCompilationEnd(J9VMThread *vmThread, TR::Compilation *comp, TR_ResolvedMet
                      J9Method *method, TR::CompilationInfoPerThreadBase *compInfoPT,
                      const std::string &codeCacheStr, const std::string &dataCacheStr)
    {
-   static const bool shouldStoreRemoteAOTMethods = !feGetEnv("TR_disableRemoteAOTMethodStorage");
+   static const bool shouldStoreRemoteAOTMethods = feGetEnv("TR_enableRemoteAOTMethodStorage");
```

Comment:
FYI it looks like this breaks the `-XX:+JITServerAOTCacheDelayMethodRelocation` option added in #17509. Since methods received from the server AOT cache are no longer stored in the local SCC (by default), after we try to delay the load by replenishing the invocation count, the next time the method reaches count 0 and is queued for compilation, there is no AOT body in the local SCC, resulting in a new remote compilation request. If it gets served from the server AOT cache again, this could even lead to an infinite loop and the method would stay interpreted.

Reply:
Yes, we should probably allow storage if that option is enabled. Either that, or ignore the option if we really don't want to store any methods in the local SCC.